src/l500/l535-amc-option.h

```diff
@@ -0,0 +1,71 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#pragma once
+#include "hw-monitor.h"
+#include "l500-device.h"
+
+namespace librealsense
+{
+    namespace ivcam2
+    {
+        namespace l535
+        {
+
+            enum amc_control
```

Comment:
Shouldn't we list only the controls that are applicable to the L535 here?

Reply:
its all of them + rx_sensitivity